checkov/common/checks_infra/solvers/attribute_solvers/any_attribute_solver.py

```diff
@@ -7,8 +7,9 @@
 class AnyResourceSolver(BaseAttributeSolver):
     operator = Operators.ANY
 
-    def __init__(self, resource_types: List[str], attribute: Optional[str], value: Any) -> None:
-        super().__init__(resource_types=resource_types, attribute=attribute, value=value)
+    def __init__(self, resource_types: List[str], attribute: Optional[str], value: Any, is_jsonpath_check: bool = False) -> None:
+        super().__init__(resource_types=resource_types, attribute=attribute, value=value,
+                         is_jsonpath_check=is_jsonpath_check)
```

Comment:
@ChanochShayner If this method does nothing, no need to overwrite the parent method